epsagon/wrappers/aws_lambda.py

```diff
@@ -94,10 +108,11 @@ def _lambda_wrapper(*args, **kwargs):
             return result
         # pylint: disable=W0703
         except Exception as exception:
-            runner.set_exception(exception, traceback.format_exc())
+            runner.set_exception(exception, traceback.format_exc(), False)
             raise
         finally:
             try:
+                _add_status_code(runner, result)
```

Comment:
It would be better if you collect it only for API-GW / load balancer trigger, and I think you should save it in the trigger.
(Now you could mistakenly collect this data when it's `metadata_only=True`)

Reply:
we agreed to allow it